modules/social_features/social_group/modules/social_group_invite/social_group_invite.install

```diff
@@ -28,3 +28,17 @@ function social_group_invite_install() {
 function social_group_invite_update_last_removed() : int {
   return 11503;
 }
+
+/**
+ * Update views bulk operations actions to make them translatable.
+ */
+function social_group_invite_update_13001(): string {
```

Comment:
@volodymyr-sydor Since this is a bug fix, and will be release in version 12.x, we need to use update hook started from `12001`

Reply:
@ribel done